.github/workflows/multiarch-build.yaml

```diff
@@ -77,4 +79,6 @@ jobs:
             podman manifest push ${{ env.IMAGE_NAME }}  ${{ env.IMAGE_REGISTRY }}/${{ env.IMAGE_NAMESPACE }}/${{ env.IMAGE_NAME }}:${{ env.IMAGE_TAG }}  --all
           elif [[ "${{ github.event_name }}" == "release" ]]; then
             podman manifest push ${{ env.IMAGE_NAME }}  ${{ env.IMAGE_REGISTRY }}/${{ env.IMAGE_NAMESPACE }}/${{ env.IMAGE_NAME }}:${{ github.ref_name }}  --all
+          elif [[ "${{ github.event_name }}" == "pull_request" ]]; then
```

Comment:
I feel like we shouldn't push images to the registry with code that hasn't been reviewed. What if we just build the image without pushing it?

Reply:
yeah, building it is minimal requirement. However, how to make easy to test PRs?

I don't want to build it locally. Maybe different org in quay?

Also, non-member cannot run workflow without approval.